docs/modules/lodash-underscore.md

```diff
@@ -0,0 +1,9 @@
+# `lodash` / `underscore`
+
+`lodash` and `underscore` have not been updated for many years and almost all methods can be replaced with modern JavaScript built-in methods, the rest - with simple helpers.
+
+Extracted from `lodash` packages like `lodash.%methodname%` are not maintained, deprecated, haven't been updated for many more years, and a part of them have significant security issues and bugs. They are no longer safe for usage.
+
+The same situation with custom `lodash` builds like `lodash-es`.
+
+You could read how to replace them, for example, here: [You don’t (may not) need Lodash/Underscore](https://you-dont-need.github.io/You-Dont-Need-Lodash-Underscore).
```

Comment:
can we put this under its own heading as a possible "replacement"?

later we can add extra headings for the various individual utils etc

Reply:
Done. I'm unsure of the format and I think we should create a template for this case.

Reply:
it'll be the same template as the others:
https://github.com/es-tooling/module-replacements/blob/main/docs/templates/module.md

but instead of an npm link, you'd only have a project link i think

Reply:
It's not sure that it's a good format here. I'll try to bring it closer.